compiler/src/iree/compiler/Codegen/Common/Passes.h

```diff
@@ -236,7 +236,8 @@ createTileAndDistributeToWorkgroupsPass(
 std::unique_ptr<Pass> createTransformDialectInterpreterPass(
     llvm::StringRef transformFileName = llvm::StringRef(),
     llvm::StringRef debugPayloadRootTag = llvm::StringRef(),
-    llvm::StringRef debugTransformRootTag = llvm::StringRef());
+    llvm::StringRef debugTransformRootTag = llvm::StringRef(),
+    llvm::StringRef transformLibraryFileName = llvm::StringRef());
```

Comment:
Would it be possible to combine the transform file name and transform library file name into a single thing? Maybe deprecate just a file name approach in favor of the file name + symbol approach. Both of these seem redundant.

Reply:
The two files have different purposes. The transformFileName will include calls to symbols defined in the library file name. Requiring them to be the same would require the user to have every transform file they pass in include the entire library. I think this concern will be addressed when switching to loading the library once so I'll keep this in mind during that refactor.

Reply:
I agree with @qedawkins having two different files makes more sense IMHO.